### PR TITLE
Update the repo url

### PR DESCRIPTION
**Pull request by stefanheyder**
We changed the Lara repository name from lara-vedst to Lara. This commit
updates all the paths (directory names and github urls).

Please have a look if these are all necessary changes, I have no prior experience with vagrant/homestead.

### CODE REVIEW

**Comment by ThomDietrich** [comment]
Hey, thank you!

1. does it have to be uppercase Lara? I would at least leave folder and repo names (technical) lowercase
2. Please do not delete the placeholder file (git reasons)

File: README.md [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
 
 ## tl;dr
 
 1. Install Git ([Windows](https://git-for-windows.github.io))
 2. Install [VirtualBox 5.1](https://www.virtualbox.org/wiki/Downloads)
 3. Install [Vagrant 1.9+](https://www.vagrantup.com/downloads.html)
 4. Open a console (Powershell, Git Bash, Terminal, ...)
- 1. `git clone https://github.com/ILSCeV/homestead-lara-vedst.git`
- 2. `cd homestead-lara-vedst`
+ 1. `git clone https://github.com/stefanheyder/homestead-Lara.git`
```
Comment (ThomDietrich) [inline README.md]: This was probably not intended!?

Comment (stefanheyder) [reply README.md]: Yes that was not indended!

**Comment by stefanheyder** [comment]
Yep we can use the lowercase name of the repo, should not matter. 
What are the git reasons? Can you fix this or should I update the pull request?

**Comment by ThomDietrich** [comment]
https://git.wiki.kernel.org/index.php/GitFaq#Can_I_add_empty_directories.3F ;)

Yes I'll take care of it. Thanks!

**Comment by ThomDietrich** [comment]
Thanks Stefan!

Everything looks fine. I've ignored the uppercase question from earlier. If "Lara" is the name of the Lara repo, the homestead repo will follow ;)

One tiny little comment: You've added tab indentation in a space-indented file. One or the other, nobody gives a damn ;) However your editor [shouldn't mix them](http://www.emacswiki.org/pics/static/TabsSpacesBoth.png). Maybe you want to check your settings.

**Comment by stefanheyder** [comment]
Thank you Thomas, I forgot to highlight tabs in my .vimrc!

**Comment by ThomDietrich** [comment]
Testing right now. Argh forget to readd the Code folder

**Comment by ThomDietrich** [comment]
@stefanheyder Everything is working again. I also had the chance to check for https://github.com/ILSCeV/Lara/issues/210 and everything is working just fine.

The changes since last time I've used Lara are also great!

I can btw recommend looking into homestead-Lara. It has numerous advantages over a local installation. Would be especially useful for the next SWP ;)

**Pull request merged**